x/storage/types/types.go

```diff
@@ -70,3 +70,22 @@ func getNFTAttributes(m interface{}) []Trait {
 	}
 	return attributes
 }
+
+func (di *DeleteInfo) IsEmpty() bool {
+	isBucketIdsEmpty := false
+	isObjectIdsEmpty := false
+	isGroupIdsEmpty := false
+	if di == nil {
+		return true
+	}
+	if di.BucketIds == nil || (di.BucketIds != nil && len(di.BucketIds.Id) == 0) {
```

Comment:
`di.BucketIds != nil` seems unnecessary.

Reply:
Because we are setting `deleteInfo. BucketIds = nil`, same for other resources, right before
```
if deleteInfo.IsEmpty() {
			deleteStalePoliciesPrefixStore.Delete(iterator.Key())
		}
```
so this check is used.

Reply:
- [ ]